### DISCUSSION TITLE
OpenRouter.ai

### DISCUSSION THREAD

**Discussion by jsalsman** [souzatharsis/podcastfy#97] [Ideas]
Would love to have OpenRouter.ai support to choose LLMs. Gemini isn't that hot in several respects. Would love to have tools like web search in particular.

**Comment by souzatharsis** [comment]
I'll enable the capability to easily swap models via langchain which we are
already integrated with.

Thanks for the feedback.

**Comment by souzatharsis** [comment]
This has been enabled now. You can use 100+ LLMs for transcript generation.

https://github.com/souzatharsis/podcastfy/blob/main/usage/how-to.md#custom-llm-support

Would love feedback, thanks!
